Server/utils/sendEmail.js

```diff
@@ -1,43 +1,38 @@
-const { MailerSend, EmailParams, Sender, Recipient } = require('mailersend')
-const logger = require('../utils/logger')
-
-
-const mailersend = new MailerSend({
-    apiKey: process.env.MAILERSEND_API_KEY,
-})
+const sgMail = require('@sendgrid/mail')
+const logger = require('./logger')
+const dotenv = require('dotenv')
+dotenv.config({path:__dirname+'/../.env'})
```

Comment:
I think this path is specifying .env should be in the root project folder?  If that's the case we should make a note in the readme to tell end user's where to put it.

I've been using the Server root folder so no config is necessary there.  Might be good to keep the server config and client config separate?

Reply:
I ran into an issue that it wasn't reading the .env file. I just figured I was testing it as a standalone, not inside the server. I will make the changes.